### PR TITLE
Mariocube.BRFNTifyNext version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103138)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230422032651

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

This package appears to require `Microsoft.VCRedist.2010.x64` (or `Microsoft.VCRedist.2010.x86`) to run.

Pending:
* https://github.com/microsoft/winget-cli/issues/163

File: manifests/m/Mariocube/BRFNTifyNext/1.0/Mariocube.BRFNTifyNext.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.BRFNTifyNext
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Wii/PC%20Applications/BRFNTify%20Next/BRFNTifyNext_B1_win32.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/m/Mariocube/BRFNTifyNext/1.0/Mariocube.BRFNTifyNext.installer.yaml:10]:
```suggestion
  NestedInstallerType: portable
```

Comment (stephengillie) [reply manifests/m/Mariocube/BRFNTifyNext/1.0/Mariocube.BRFNTifyNext.installer.yaml:10]: In addition to the dependency, this package needs to be a `portable` type.

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
Seems the package flags don't apply retroactively. 
[Policy] Reset Feedback

**Pull request closed without merging**